src/test/java/uk/gov/pay/directdebit/payments/resources/PaymentSearchResourceIT.java

```diff
@@ -34,20 +32,25 @@
 import static uk.gov.pay.directdebit.payments.fixtures.GatewayAccountFixture.aGatewayAccountFixture;
 import static uk.gov.pay.directdebit.payments.fixtures.PaymentFixture.aPaymentFixture;
 
-@RunWith(DropwizardJUnitRunner.class)
-@DropwizardConfig(app = DirectDebitConnectorApp.class, config = "config/test-it-config.yaml")
-public class PaymentSearchResourceIT {
+public class PaymentSearchResourceIT { //TODO merge with PaymentResource
```

Comment:
can this spurious comment `//TODO merge with PaymentResource` be removed please. i think we don't want to merge it with PaymentResource after all.